free-programming-books-hu.md

```diff
@@ -22,7 +22,7 @@
 * [Az adatbázistervezés alapjai és titkai](http://mek.oszk.hu/11100/11123/index.phtml) - Halassy Béla (Word, PDF)
 * [Ember, információ, rendszer](http://mek.oszk.hu/11100/11122/index.phtml) - Halassy Béla (Word, PDF)
 * [Formális nyelvek](http://mek.oszk.hu/05000/05099/index.phtml) - Bach Iván (PDF)
-* [Kanban és Scrum mindkettőből a legjobbat](http://www.adaptiveconsulting.hu/dokumentumok) - Henrik Kniberg, Mattias Skarin, ford.: Csutorás Zoltán és Marhefka István (PDF)
+* [Kanban és Scrum mindkettőből a legjobbat](http://www.adaptiveconsulting.hu) - Henrik Kniberg, Mattias Skarin, ford.: Csutorás Zoltán és Marhefka István (PDF)
```

Comment:
Done in https://github.com/davisonio/awesome-free-programming-books/commit/9d81cbc6bf6251935fd3f88df59ae66843e2f42e